src/PickerContext/index.js

```diff
@@ -49,7 +49,22 @@ PickerContextProvider.propTypes = {
 };
 
 export function useActiveCategory() {
-  return useContext(PickerContext).activeCategoryState;
+  const [activeCategory] = useContext(PickerContext).activeCategoryState;
+  return activeCategory;
+}
+
+export function useSetActiveCategory() {
+  const [, setActiveCategory] = useContext(PickerContext).activeCategoryState;
+  const [, setSeenGroups] = useContext(PickerContext).seenGroupsState;
+
+  return categoryName => {
+    setActiveCategory(categoryName);
+    setSeenGroups((seenGroups = {}) => {
+      return seenGroups[categoryName]
+        ? seenGroups
+        : { ...seenGroups, [categoryName]: true };
+    });
+  };
```

Comment:
This exact logic is already implemented in `useSetSeenGroups`
```suggestion
  setSeenGroups = useSetSeenGroups();

  return categoryName => {
    setActiveCategory(categoryName);
    setSeenGroups(categoryName);
  };
```